pkg/internal/codegen/parse/crd.go

```diff
@@ -470,6 +473,30 @@ func (b *APIs) parseObjectValidation(t *types.Type, found sets.String, comments
 	return props, buff.String()
 }
 
+// parseNativeObjectValidation returns a JSONSchemaProps object and its serialization in
+// Go that describe the validations for the given native object type.
+func (b *APIs) parseNativeObjectValidation(t *types.Type, found sets.String, comments []string, isRoot bool) (v1beta1.JSONSchemaProps, string) {
+	buff := &bytes.Buffer{}
+	props := v1beta1.JSONSchemaProps{
+		Type:        "object",
+		Description: parseDescription(comments),
+	}
+
+	for _, l := range comments {
```

Comment:
How about `c` for comment?

Reply:
The new method is just copy-paste of `parseObjectValidation` without the check for `strings.HasPrefix(t.Name.String(), "k8s.io/api")`.

Reply:
Don't like this special casing of ObjectMeta. Our mechanisms should work in the general case.